include/ignition/math/eigen3.hh

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
I would have expected this file in the `eigen/include/ignition/math` folder

Reply:
Updated! https://github.com/ignitionrobotics/ign-math/pull/413/commits/aca4520863e633ea934a796e51a056c7606510d0